docs/settings.md

```diff
@@ -0,0 +1,204 @@
+# Settings
+
+This page will cover the Settings sections.  
+Each setting type is grouped based on what they're related to.
+
+## Media Management
+
+Here is where you configure your settings related to how your media files are handled.
+
+### Naming
+
+#### Volume Folder Naming
+
+The naming format for the volume folder (folder structure allowed).  
+Available variables are:
+
+- {series_name}
+- {clean_series_name}
+- {volume_number}
+- {comicvine_id}
+- {year}
+- {publisher}
+
+For example, `{series_name} ({year})` will come out as "Invincible Iron Man (2019)".
+
+#### File Naming
+
+The naming format for the file itself.  
+Available variables are:
+
+- {series_name}
+- {clean_series_name}
+- {volume_number}
+- {comicvine_id}
+- {year}
+- {publisher}
+- {issue_comicvine_id}
+- {issue_number}
+- {issue_title}
+- {issue_release_date}
+- {issue_release_year}
+
+For example, `{series_name} ({year}) Volume {volume_number} Issue {issue_number}` would come out as "Invincible Iron Man (2019) Volume 01 Issue 001.cbz".
+
+#### File Naming For TPB
+
+The naming format for the file itself (if it's a TPB).  
+Available variables are:
+
+- {series_name}
+- {clean_series_name}
+- {volume_number}
+- {comicvine_id}
+- {year}
+- {publisher}
+
+For example, `{series_name} ({year}) Volume {volume_number} TPB` would come out as "Invincible Iron Man (2019) Volume 01 TPB.cbz"
+
```

Comment:
It seems like the File Naming For Issues Without Title and Treat Volume Issues As "No Title" are missing.

Reply:
Added